src/clinicaltrialsgov/index.js

```diff
@@ -391,13 +371,20 @@ const upload = async ({ conn }) => {
                 pageSize: 1000,
                 pageToken: next,
                 'query.cond': 'cancer',
+                sort: 'LastUpdatePostDate',
+                ...options,
             },
             uri: BASE_URL,
         });
 
         for (const trial of trials.studies) {
+            if (processCount > total) {
+                break;
+            }
+
             try {
                 const record = convertAPIRecord(trial);
+                logger.info(`processing (${processCount++}/${total}) record: ${record.sourceId}`);
```

Comment:
I think we shoul increment processCount on a seperate line.

Reply:
Applied.